src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -38,6 +38,12 @@ const DraggableComponent: FC<draggableProps> = ({
     title = '',
 }) => {
     const draggableIds = useContext(disableDrag);
+        const { data: userResponse } = useGetUsersByUsernameQuery({
+        searchString: draggableId,
+        size: 1,
```

Comment:
What do 1 refer?

Reply:
The number **1** in the code refers to the size parameter which return only one first response and  **draggableId** is the username parameter passed to the `useGetUsersByUsernameQuery `hook. This hook appears to be part of an API service created using the @reduxjs/toolkit library.

Reply:
But if a new developer comes and checks the code they won't get this right? Better to store it in variable

Reply:
doen i create the new constants **MAX_SEARCH_RESULTS** and set value 1 for **size**.